examples/authenticated-api/echo/api/api.gen.go

```diff
@@ -288,7 +288,8 @@ type ClientWithResponsesInterface interface {
 type ListThingsResponse struct {
 	Body         []byte
 	HTTPResponse *http.Response
-	JSON200      *[]ThingWithID
+
+	JSON200 *[]ThingWithID
```

Comment:
Having a bit of difficulty tweaking the whitespace trimming in the template to avoid this diff noise 🤔

Reply:
It is because of the if-statements you introduced. Their tempalte closing tags are `}}`, while `-}}` would prevent the new lines (I guess, didn't try it)